docs/source/index.rst

```diff
@@ -108,7 +108,7 @@ table.
 +----------------------------------+-----------------------------------------------------------------------------------------------------+
 | Component                        | Citation                                                                                            |
 +==================================+=====================================================================================================+
-| Celular Automaton                | `Tucker et al. (2015) <https://www.geosci-model-dev.net/9/823/2016/>`_                              |
+| Cellular Automaton                | `Tucker et al. (2015) <https://www.geosci-model-dev.net/9/823/2016/>`_                              |
```

Comment:
Since there is now one more character, this table needs to be modified (its why the doc stage failed on one of the travis builds).

Reply:
whoops. fixed.